3574/2/helpers/photo.rb

```diff
@@ -0,0 +1,19 @@
+require 'open-uri'
+require 'json'
+
+BOT_PATH = "https://api.telegram.org/bot#{ENV['TOKEN']}/".freeze
+BOT_PATH_FOR_DOWNLOAD = "https://api.telegram.org/file/bot#{ENV['TOKEN']}/".freeze
```

Comment:
I'd remove slashes from string endings and use them(slashes) inside string interpolation